lib/api/views.js

```diff
@@ -41,7 +41,7 @@ const viewPath = view => path.resolve(__dirname, '..', '..', 'views', (view || '
  * @param  {Function} callback Optional callback function.
  * @return {String}            Renders and returns the template HTML.
  */
-const render = (data, req, callback) => {
+const render = (data, req, res, callback) => {
```

Comment:
@smebberson I thought the callback could be a `res` object or `callback` function. Do you remember how it was implemented?

Reply:
@allanchau, you're correct. However, it seems as though `getScripts` and `getStyles` requires `res`. See L59 and L60.

Reply:
I think we should keep this change, so that it requires `res`, but update the code further below so that callback is an optional argument. It won't take much.